tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -1184,11 +1145,32 @@ handle_build() {
 			tags+=("$(cat "${version_file}")")
 		fi
 
+		echo "Pushing ${build_target} with tags: ${tags[*]}"
+
 		for tag in "${tags[@]}"; do
 			case ${build_target} in
+				kernel-nvidia-gpu)
+					sudo oras push \
+						${ARTEFACT_REGISTRY}/kata-containers/cached-artefacts/${build_target}:latest-${tag} \
```

Comment:
Hi @stevenhorsman !

This will tag like `latest-latest-${TARGET_BRANCH}-$(uname -m)`  (prefixed `latest` twice).

Could you give an example of `artefact_tag`?

Reply:
Ah goid spot. I'll fix that. The example `artefact_tag`s we have are:
- agent - this has the short hash of the kata-containers repo
- pause image - this is the <repo>-<tag> of the pause image. e.g. `docker___registry.k8s.io_pause-3.9` once we replaced the not allow characters
- guest-components - the version.yaml for guest components e.g. `adca2f94091d73c0b0c96a7789322a801c15811b` or `v0.9.0`

Does that help?

Reply:
Yes, it does help. Thanks Steve!